azurerm/resource_arm_databricks_workspace.go

```diff
@@ -0,0 +1,244 @@
+package azurerm
```

Comment:
can we also add some documentation for this resource to the `website` folder ([example](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/dns_a_record.html.markdown)) and to [the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L248)?